client/tailscale/localclient.go

```diff
@@ -825,6 +829,25 @@ func (lc *LocalClient) GetDNSOSConfig(ctx context.Context) (*apitype.DNSOSConfig
 	return &osCfg, nil
 }
 
+// PerformDNSQuery exexutes a DNS query for the given name and query type. It returns the
+// raw DNS response bytes and the resolvers that were used to answer the query (often just
+// one, but can be more if we raced multiple resolvers).
+func (lc *LocalClient) PerformDNSQuery(ctx context.Context, name string, queryType dnsmessage.Type) (bytes []byte, resolvers []*dnstype.Resolver, err error) {
+	body, err := lc.get200(ctx, fmt.Sprintf("/localapi/v0/dns-query?name=%s&type=%d", url.QueryEscape(name), queryType))
+	if err != nil {
+		return nil, nil, err
+	}
+	type response struct {
+		Bytes     []byte
+		Resolvers []*dnstype.Resolver
+	}
```

Comment:
don't we have localapi types in apitype or somewhere?  would be nice to unify this and then have docs on them, even if they're internal

Reply:
done